clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java

```diff
@@ -166,6 +166,12 @@
                                                        + " values are <code>none</code>, <code>gzip</code>, <code>snappy</code>, <code>lz4</code>, or <code>zstd</code>. "
                                                        + "Compression is of full batches of data, so the efficacy of batching will also impact the compression ratio (more batching means better compression).";
 
+    /** <code>compression.level</code> */
+    public static final String COMPRESSION_LEVEL_CONFIG = "compression.level";
+    private static final String COMPRESSION_LEVEL_DOC = "The compression level for all data generated by the producer. The default level and valid value is up to "
+                                                        + "compression.type. (<code>none</code>, <code>snappy</code>: not available. <code>gzip</code>: 1~9 (default: 1). "
+                                                        + "<code>lz4</code>: 4~7 (default: 4). <code>zstd</code>: 1~22 (default: 3).";
```

Comment:
What about negative (fast) ones for zstd?

Reply:
I will be updated with zstd-jni-1.3.7-2 accordingly. It was just released about 4 hours from now.

Reply:
@dongjinleekr , yes, I noticed that you also want it and I pushed a release.